optimize/util/optimize-commons/src/test/java/io/camunda/optimize/service/db/schema/OptimizeIndexNameServiceTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.db.schema;
+
+import static io.camunda.optimize.service.db.DatabaseConstants.COMPONENT_NAME;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import io.camunda.optimize.service.db.es.schema.index.index.PositionBasedImportIndexES;
+import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class OptimizeIndexNameServiceTest {
+
+  @ParameterizedTest
+  @ValueSource(strings = {"", "my-index"})
+  public void shouldReturnIndexWithPrefixAndComponent(final String indexPrefix) {
+    final var nameService = new OptimizeIndexNameService(indexPrefix);
+
+    assertThat(
+            nameService.getOptimizeIndexOrTemplateNameForAliasAndVersionWithPrefix(
+                "position-index", "8.3"))
+        .isEqualTo(withHyphen(indexPrefix) + "optimize-position-index_v8.3");
```

Comment:
not a big deal, but the Optimize index versions are just integers (unrelated to the actual project version), so lets reflect that in this test. Concretely, just use `v8` rather than `v8.3`